src/routes/admin.ts

```diff
@@ -1,11 +1,12 @@
 import { Router } from 'express';
-import { adminLogin, getTherapistsForAdmin } from '../controllers';
+import { adminLogin, getTherapistsForAdmin, updateTherapistActive } from '../controllers';
 import { checkAuth } from '../middlewares';
 import { RolesForSelect } from '../types';
 
 const router = Router();
 
 router.post('/login', adminLogin);
 router.get('/therapists', checkAuth(RolesForSelect.admin), getTherapistsForAdmin);
+router.patch('/therapists/:id/:active', checkAuth(RolesForSelect.admin), updateTherapistActive);

```

Comment:
not sure why active is a request params, please send it in request body, and maybe we can add therapistId as well